metricbeat/helper/sql/sql.go

```diff
@@ -144,7 +147,8 @@ func (d *DbClient) fetchVariableMode(rows sqlRow) (mapstr.M, error) {
 	r := mapstr.M{}
 
 	for key, value := range data {
-		value := getValue(&value)
+		value := value
```

Comment:
What do you think about renaming `value` inside the loop to avoid redeclaration and confusion?

Reply:
Had to fix it because it was pointed out by the linter.

> In Go, variables introduced in a for statement are "re-used in each iteration" ([see the specification](https://golang.org/ref/spec#For_statements)). This means taking the address of one of these variable always produces a pointer to the same location, which is re-used on each iteration

There's an analyzer in `go vet` as well that does checks for this. Also, refer: https://garbagecollected.org/2017/02/22/go-range-loop-internals/

With:
```
value := value
```
we now have a copy that can safely be used.